tests/operator/operator.go

```diff
@@ -767,14 +767,11 @@ var _ = Describe("[Serial][sig-operator]Operator", Serial, decorators.SigOperato
 
 		generateNonMigratableVMIs = func(num int) []*v1.VirtualMachineInstance {
 
-			vmis := []*v1.VirtualMachineInstance{}
+			vmis := make([]*v1.VirtualMachineInstance, num)
 			for i := 0; i < num; i++ {
-				vmi := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskCirros))
-				// Remove the masquerade interface to use the default bridge one
-				// bridge interface isn't allowed to migrate
-				vmi.Spec.Domain.Devices.Interfaces = nil
-				vmi.Spec.Networks = nil
-				vmis = append(vmis, vmi)
+				vmis[i] = libvmi.New(libvmi.WithNamespace(util2.NamespaceTestDefault),
+					libvmifact.WithArchAwareMemory(),
+					libvmi.WithContainerDisk("disk0", cd.ContainerDiskFor(cd.ContainerDiskCirros)))
 			}
```

Comment:
Please drop this helper lambda and just call it twice:
```go
vmisNonMigratable := []*v1.VirtualMachineInstance(
    libvmi.New(...),
    libvmi.New(...),
)
```
If you want to optimize further, you could first collect the options and then use the same ones to create the VMI, but it is not really needed IMO.